docs/operating-eck/troubleshooting/common-problems.asciidoc

```diff
@@ -84,3 +84,75 @@ type: Opaque
 ----
 
 Failure to do so can cause data loss.
+
+[id="{p}-{page_id}-pod-updates"]
+== Pods are not replaced after configuration update
+
+When updating an existing Elasticsearch cluster configuration, the update can fail while being be applied.  This happens when the operator is unable to apply the changes required by replacing the pods of a given the Elasticsearch cluster.
+
+This can be identified by the `AGE` of the pod being older than the configuration update or the `STATUS` of the pods
+
+[source,sh]
+----
+kubectl get pods
+
+NAME                      READY   STATUS    RESTARTS   AGE
+quickstart-es-default-0   1/1     Running   0          15m
+quickstart-es-default-1   1/1     Running   0          15m
+quickstart-es-default-2   1/1     Running   0          15m
+----
+
+The causes include
+
+- Elasticsearch cluster is not healthy
+
+The health of the cluster may look like
+
+[source,sh]
+----
+kubectl get elasticsearch
+
+NAME         HEALTH   NODES   VERSION   PHASE   AGE
+TODO: Finish example
+----
+
+TODO:
+Action: Check and fix your shard allocations.
+(more API details can be added here)
+For more information, see https://www.elastic.co/guide/en/elasticsearch/reference/7.10/cluster-health.html
+and https://www.elastic.co/guide/en/cloud-on-k8s/1.3/k8s-deploy-elasticsearch.html#k8s_monitor_cluster_health_and_creation_progress.
+
+- Not enough resources in the Kubernetes cluster
+
+The scheduling may fail with a message such as
+
+[source,sh]
+----
+kubectl get events --sort-by='{.lastTimestamp}' | tail
+
+LAST SEEN   TYPE      REASON             OBJECT                        MESSAGE
+10s         Warning   FailedScheduling   pod/quickstart-es-default-2   0/3 nodes are available: 3 Insufficient memory.
+----
+
+Or a pod is stuck in `STATUS` of `Pending` with an operator log of unable to `restart some nodes`
```

Comment:
We might reword this sentence. It's a bit unclear...

Reply:
I took a stab at a rewording it, does this help?